examples/grid.py

```diff
@@ -0,0 +1,62 @@
+import pygame
+
+TITLE = "Grid"
+TILES_HORIZONTAL = 10
+TILES_VERTICAL = 10
+TILE_SIZE = 128
+WINDOW_WIDTH = 1280
+WINDOW_HEIGHT = 1280
+
+
+class Player:
+    def __init__(self, surface):
+        self.surface = surface
+        self.pos = (64, 64)
+
+    def draw(self):
+        pygame.draw.circle(self.surface, (255, 255, 255), self.pos, 64)
+
+    def move(self, target):
+        x = (128 * (target[0] // 128)) + 64
+        y = (128 *(target[1] // 128)) + 64
+
+        self.pos = (x, y)
+
+
+class Game:
+    def __init__(self):
+        pygame.init()
+        self.clock = pygame.time.Clock()
+        pygame.display.set_caption(TITLE)
+        self.surface = pygame.display.set_mode((WINDOW_WIDTH, WINDOW_HEIGHT))
+        self.loop = True
+        self.player = Player(self.surface)
+
+    def main(self):
+        while self.loop:
```

Comment:
I think it's better to merge the two functions `events()` and `draw()` together.